### ISSUE TITLE
Can not obtain the results reported in paper

### ISSUE THREAD

**Issue by zhou-wj** [PrekshaNema25/DiverstiyBasedAttentionMechanism#1]
I wonder how many epoch you have run to obtain the results you reported in your paper?
I ran the source code with your default parameters, **### after 6 epochs, the loss decrease to nan**, please check the information below for details of the output:

> Epoch: 6
Step 299: loss = 0.16 (88.490 sec)
Training Data Eval:
Predicted Summary is  big law lawyers can t have much of a life
True Summary is  big law lawyers can t have much of a life
Validation Data Eval:
Predicted Summary is  many women an lasting of assault from man <eos> can
True Summary is  contraception use leads to the degradation of women and their
Step 599: loss = 0.15 (177.473 sec)
Training Data Eval:
Predicted Summary is  holocaust denial cannot be criminal on basis of protecting israel
True Summary is  holocaust denial cannot be criminal on basis of protecting israel
Validation Data Eval:
Predicted Summary is  israel s collective of meet obligation for gazans <eos> <eos>
True Summary is  israel blockade allows humanitarian aid into gaza <eos> <pad> <pad>
Step 899: loss = nan (266.051 sec)
Training Data Eval:
Predicted Summary is  <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad>
True Summary is  coal electricity still cost competitive even after ccs costs <eos>
Validation Data Eval:
Predicted Summary is  <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad>
True Summary is  <unk> to alter with drug use the god given human
Step 1199: loss = nan (354.456 sec)
Training Data Eval:
Predicted Summary is  <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad>
True Summary is  inconsistent wind energy in pickens plan can be <unk> <eos>
Validation Data Eval:
Predicted Summary is  <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad>
True Summary is  customers will pay more attention to calorie counts over time
Step 1499: loss = nan (443.324 sec)
Training Data Eval:
Predicted Summary is  <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad>
True Summary is  of geoengineering could be worse than climate change <pad> <pad>
Validation Data Eval:
Predicted Summary is  <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad>
True Summary is  <unk> union demonstrates failures of socialism <eos> <pad> <pad> <pad>
Step 1763: loss = nan (521.101 sec)
Training Data Eval:
Predicted Summary is  <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad>
True Summary is  tax rebates do not always lead to increased spending stimulation
Validation Data Eval:
Predicted Summary is  <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad>
True Summary is  targeted assassinations disrupt and reduce terrorist attacks <eos> <pad> <pad>
Training Loss:nan
Validation Loss:nan
Total time:530.240172624588
Epoch: 7
Step 299: loss = nan (88.202 sec)
Training Data Eval:
Predicted Summary is  <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad>
True Summary is  marriage is safety net for <unk> pregnancies n a for
Validation Data Eval:
Predicted Summary is  <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad> <pad>
True Summary is  contraception use leads to the degradation of women and their

I ran the model with the split dataset 1, 2, and 3, **### but the results is far below your reported results**, do I need to preprocess the data such as tokenize or delete stop words?
The results post below:

> UGE-1.5.5.pl -e /home/zmh/install/rouge/RELEASE-1.5.5/data -c 95 -2 -1 -U -r 1000 -n 4 -w 1.2 -a -m /tmp/tmpMyy6wn/rouge_conf.xml
---------------------------------------------
1 ROUGE-1 Average_R: 0.20425 (95%-conf.int. 0.19512 - 0.21395)
1 ROUGE-1 Average_P: 0.20244 (95%-conf.int. 0.19260 - 0.21226)
1 ROUGE-1 Average_F: 0.19984 (95%-conf.int. 0.19045 - 0.20905)
---------------------------------------------
1 ROUGE-2 Average_R: 0.04296 (95%-conf.int. 0.03513 - 0.05106)
1 ROUGE-2 Average_P: 0.04301 (95%-conf.int. 0.03476 - 0.05139)
1 ROUGE-2 Average_F: 0.04207 (95%-conf.int. 0.03439 - 0.05008)
---------------------------------------------
1 ROUGE-3 Average_R: 0.02334 (95%-conf.int. 0.01607 - 0.03082)
1 ROUGE-3 Average_P: 0.02300 (95%-conf.int. 0.01592 - 0.03048)
1 ROUGE-3 Average_F: 0.02293 (95%-conf.int. 0.01586 - 0.03018)
---------------------------------------------
1 ROUGE-4 Average_R: 0.01961 (95%-conf.int. 0.01283 - 0.02671)
1 ROUGE-4 Average_P: 0.01923 (95%-conf.int. 0.01245 - 0.02609)
1 ROUGE-4 Average_F: 0.01930 (95%-conf.int. 0.01263 - 0.02620)
---------------------------------------------
1 ROUGE-L Average_R: 0.18332 (95%-conf.int. 0.17419 - 0.19246)
1 ROUGE-L Average_P: 0.18168 (95%-conf.int. 0.17238 - 0.19085)
1 ROUGE-L Average_F: 0.17921 (95%-conf.int. 0.17033 - 0.18817)
---------------------------------------------
1 ROUGE-W-1.2 Average_R: 0.11404 (95%-conf.int. 0.10835 - 0.12016)
1 ROUGE-W-1.2 Average_P: 0.17178 (95%-conf.int. 0.16315 - 0.18066)
1 ROUGE-W-1.2 Average_F: 0.13394 (95%-conf.int. 0.12738 - 0.14071)
---------------------------------------------
1 ROUGE-S* Average_R: 0.04736 (95%-conf.int. 0.03998 - 0.05519)
1 ROUGE-S* Average_P: 0.04680 (95%-conf.int. 0.03919 - 0.05487)
1 ROUGE-S* Average_F: 0.04477 (95%-conf.int. 0.03772 - 0.05256)
---------------------------------------------
1 ROUGE-SU* Average_R: 0.07662 (95%-conf.int. 0.06893 - 0.08505)
1 ROUGE-SU* Average_P: 0.07586 (95%-conf.int. 0.06807 - 0.08424)
1 ROUGE-SU* Average_F: 0.07244 (95%-conf.int. 0.06489 - 0.08034)

Thanks for your help
